include/baresip.h

```diff
@@ -689,6 +690,7 @@ const char *menc_event_name(enum menc_event event);
 struct network;
 
 typedef void (net_change_h)(void *arg);
+typedef int (net_laddr_h)(const struct sa *sa);
```

Comment:
perhaps also add interface name ?

Reply:
Yes, this makes sense.

Reply:
But if we drop the net->laddrs list, then this function pointer is obsolete.

Reply:
I tried to add the interface name to `struct laddr`. But this enlarges the PR. And the benefit is low.